notify.go

```diff
@@ -435,6 +519,115 @@ func parseChainNtfnParams(params []json.RawMessage) (*chainhash.Hash,
 	return blockHash, blockHeight, blockTime, nil
 }
 
+// parseFilteredBlockConnectedParams parses out the parameters included in a
+// filteredblockconnected notification.
+//
+// NOTE: This is a btcd extension ported from github.com/decred/dcrrpcclient
+// and requires a websocket connection.
+func parseFilteredBlockConnectedParams(params []json.RawMessage) (int32,
+	*wire.BlockHeader, []*btcutil.Tx, error) {
+
+	if len(params) != 3 {
```

Comment:
We should avoid the same problem we ran into with extending `blockconnected` by instead of checking for != 3, check `if len(params) < 3 {`.  That way it can be extended.  I realize the rest of the code is currently !=, but I think the problem we ran into here shows we should move to using <.